src/hooks/getInfoResponse.ts

```diff
@@ -0,0 +1,12 @@
+export const getInfoResponse = (id: string) =>
+  fetch(`${id.replace(/\/$/, "")}/info.json`)
+    .then((response) => response.json())
+    .then((json) => json)
+    .catch((error: any) => {
+      console.error(
```

Comment:
Just thinking if this is a hook, and it enters the `catch` block if we should provide a reasonable error response to calling code.  Like `null` or `{}` or something maybe?